pyproject.toml

```
@@ -44,7 +44,7 @@ max-module-lines = 1000
 [tool.poetry.dependencies]
 python = "^3.8"
 boto3 = "^1.26.60"
-kubernetes = "^25.3.0"
+kubernetes = "^24"
```

https://github.com/kubernetes-client/python/#compatibility-matrix-of-supported-client-versions
```suggestion
kubernetes = ">=22,<25"
```